manifest_template.yml

```diff
@@ -66,10 +78,10 @@ apigee:
 {% set portal_visibility = ENV.portal_visibility | default(true) %}
 {% set TITLE = 'e-Referral Service' %}
 {% set NAME = 'e-referrals-service-api-' + ENV.name %}
-{% if ENV.name == 'internal-dev' %}
+{% if ENV.has_variants == 'true' %}
   - name: {{ ENV.name }}
```

Comment:
I tried {% if ENV.variants %} which should just check for the existance of the variants array - but it failed in the APIM utils manifest validation, hence I had to put in the has_variants for all the ENVs